src/default/assets/js/src/typedoc/services/Viewport.ts

```diff
@@ -23,12 +28,31 @@ namespace typedoc
          */
         height:number = 0;
 
+        /**
+         * The toolbar (contains the search input).
+         */
+        toolbar:HTMLDivElement;
+
+        /**
+         * Boolean indicating whether the toolbar is shown.
+         */
+        showToolbar:boolean = true;
+
+        /**
+         * The toolbar (contains the search input).
```

Comment:
Is this the toolbar or the `toolbar` property above?

Reply:
Copy paste error!